Packs/CommunityCommonDashboards/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Community Common Dashboards",
     "description": "A pack that contains community dashboards",
     "support": "community",
-    "currentVersion": "2.0.1",
+    "currentVersion": "2.1.0",
```

```suggestion
    "currentVersion": "2.0.2",
```

Please update also the release notes file name accordingly